integration/cypress/integration/legacy/dns.spec.ts

```diff
@@ -1,27 +1,27 @@
-import { generateLegacyURL } from "@maas-ui/maas-ui-shared";
+import { generateNewURL } from "@maas-ui/maas-ui-shared";
 
```

Comment:
Should this file get moved out of `/legacy/`?

Reply:
Indeed!